### PR TITLE
pretest commit of documentation for journals issn

### PR DESCRIPTION
**Pull request by seasidesparrow**
modified:   API_documentation_Python/Journals_API_Python.ipynb
 	modified:   openapi/openapi.yaml
 	modified:   openapi/openapi_public.yaml
 	modified:   openapi/parameters.yaml
 	modified:   openapi/schemas.yaml
 	modified:   openapi/services/journals.yaml

### CODE REVIEW

File: API_documentation_Python/Journals_API_Python.ipynb [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "* `/journal/<search text>`:generate a list of bibstems whose journal name and abbreviation data contain `<search text>`; `<search text>` must be url-encoded text\n",
     "* `/refsource/<bibstem>`: generate a list of citation data sources on a per volume basis for a given `<bibstem>`\n",
     "* `/holdings/<bibstem>/<volume>`: generate a list of available electronic sources for a given `<bibstem>/<volume>`\n",
+    "* `/issn/<search text>`: return the ADS bibstem for the journal having an ISSN identifier that matches `<search text>`\n",
```
Comment (kelockhart) [inline API_documentation_Python/Journals_API_Python.ipynb:17]: Can you add this and the example to the UNIX shell notebook as well? https://github.com/adsabs/adsabs-dev-api/blob/master/API_documentation_UNIXshell/Journals_API.ipynb

**Review by kelockhart — CHANGES_REQUESTED** [review]
Looks good! Can you also add the new endpoint and example to the UNIX shell Jupyter notebook? Other than that, it's good to go.

File: API_documentation_Python/Journals_API_Python.ipynb (hunk 1 above)
Comment (seasidesparrow) [reply API_documentation_Python/Journals_API_Python.ipynb:17]: This is done, with the caveat that I can't access the endpoint so I can't get real numbers back for the stdout transfer rate.  If that isn't a big deal, I think it's ready to go.

**Review by kelockhart — APPROVED** [review]
Looks good! Merge when ready.

**Pull request merged**